packer/scylla.json

```diff
@@ -4,40 +4,6 @@
       "name": "aws",
       "type": "amazon-ebs",
       "access_key": "{{user `access_key`}}",
-      "ami_block_device_mappings": [
```

Comment:
how this is related to the snapshot size ?

Reply:
It's not been used. so I removed it. I can update the commit message or split it to different commit